lambdas/service/app.py

```diff
@@ -183,36 +183,54 @@ def openapi():
 
 health_spec = {
     'responses': {
-        '200': {
-            'description': '200 response',
+        f'{200 if up else 503}': {
+            'description': f'Service component is {"" if up else "un"}reachable.',
             'content': {
                 'application/json': {
-                    'schema': {'$ref': '#/components/schemas/Empty'}
+                    'schema': schema.object(
+                        additional_properties=schema.object(
+                            additional_properties=True,
+                            up=schema.enum(up)
+                        ),
+                        up=schema.enum(up)
+                    )
                 }
             }
-        }
+        } for up in [True, False]
     },
     'tags': ['Auxiliary']
 }
 
 
 @app.route('/health', methods=['GET'], cors=True, method_spec={
     **health_spec,
-    'summary': 'List health information for webservice'
+    'description': 'List health information for Azul service',
+    'summary': 'Check Azul health'
 })
 @app.route('/health/{keys}', methods=['GET'], cors=True, path_spec={
     'parameters': [
         {
             'name': 'keys',
```

Comment:
This parameter needs a description. The fact that `keys` is a comma separated list of these enum values isn't captured anywhere. Maybe an enum isn't the right way to spec this. Consider restricting by regex pattern like `^(api_endpoints|basic|cached|…)(,(api_endpoints|basic|cached|…))*$`